packages/core/contracts/financial-templates/test/MockFundingRateApplier.sol

```diff
@@ -0,0 +1,29 @@
+pragma solidity ^0.6.0;
+
+pragma experimental ABIEncoderV2;
+
+import "../common/FundingRateApplier.sol";
+
+
+// A mock funding rate applier used for testing.
+contract MockFundingRateApplier is FundingRateApplier {
```

Comment:
tiny tiny nit: is this a mock? I'm actually not sure what the exact definition of a mock is (so I totally could be wrong), but elsewhere we've just called these contracts `FundingRateApplierTest` rather than a mock since they inherit from and give access to the internals/private functions of a production contract rather than "faking" or "mocking" its behavior for use by other contracts.

See https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/oracle/implementation/test/ResultComputationTest.sol and https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/oracle/implementation/test/GovernorTest.sol.

Reply:
Good call, it's not a mock its just a test wrapper contract